src/DreamCommerce/Component/Common/Model/ArrayableTrait.php

```diff
@@ -10,8 +10,9 @@
 
 namespace DreamCommerce\Component\Common\Model;
 
-use ArrayAccess;
-use DateTime;
+use \ArrayAccess;
+use \DateTime;
```

Comment:
Jak używasz use to nie zaczynaj od /. Jakbyś uruchomił code sniffera: composer cs-fix to powinien on to naprawić